spec/integration/v2/rules_spec.rb

```diff
@@ -355,7 +356,7 @@
 
       v2_auth_header
       tags 'Policies'
-      description 'Unassigns a Rule from a Tailoring'
+      description 'Customizing of the tailoring by removing rule'
```

Comment:
I think is a little confusing. Change to:
Use this to remove a rule from your tailoring.